src/styles/main.scss

```diff
@@ -1,3 +1,43 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+  box-sizing: border-box;
+}
+
+@import "./utils/variables.scss";
+
 body {
   margin: 0;
 }
+
```

Comment:
It would be better to create _reset.scss

Reply:
In general, @strukk311 is right, but in this case, additional file isn't needed